pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -45,13 +45,15 @@ def __init__(self,
                  namespace=None,
                  worker_id=None,
                  trigger_kind=None,
-                 trigger_name=None):
+                 trigger_name=None,
+                 decode_events=True):
```

Comment:
```suggestion
                 decode_strings=True):
```

Reply:
well, obviously we decode strings here, but that alone doesnt tell what _strings_ it decodes.